olca-app/src/org/openlca/app/tools/graphics/themes/Theme.java

```diff
@@ -25,18 +26,19 @@ public class Theme {
 	private boolean isDark;
 	private Color backgroundColor;
 	private Color defaultLinkColor;
-	private Color defaultLinkColorSelection;
 	private Color infoLabelColor;
 
 	private final EnumMap<FlowType, Color> flowLabelColors;
 	private final EnumMap<FlowType, Color> linkColors;
+	private final EnumMap<Css.PseudoElement, Color> linkPseudoColors;
```

Comment:
also here, we have `linkColors` and `linkPseudoColors`, isn't the first just for the graph editor and the second for the Sankey graph? so then `sankeyLinkColors` would be directly clear and should be used, it is super confusing otherwise.

Reply:
My idea is that we use `::` in the CSS which is called pseudo-element. And I agree with you, it can be specific to Sankey.